src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -6,7 +6,7 @@
     private static final int DEFAULT_CAPACITY = 10;
     private static final String BOUNDS_EXCEPTION = "Out of bounds for index ";
     private static final String NO_SUCH_ELEMENT_EXCEPTION = "No such element ";
-    private int currentIndex = 0;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
primitives have default values